torchgeo/trainers/byol.py

```diff
@@ -449,3 +452,16 @@ def validation_step(self, *args: Any, **kwargs: Any) -> None:
 
     def test_step(self, *args: Any, **kwargs: Any) -> Any:
         """No-op, does nothing."""
+
+    def predict_step(self, *args: Any, **kwargs: Any) -> Tensor:
+        """Compute and return the output embeddings of the image encoder.
+
+        Args:
+            batch: the output of your DataLoader
+        Returns:
+            image embeddings
+        """
+        batch = args[0]
+        x = batch["image"]
+        _ = self(x)
```

Comment:
```suggestion
        self(x)
```

Reply:
I believe this would print the outputs which is not what we want.

Reply:
I was wrong, it doesn't seem to print the outputs.

Reply:
It isn't a Jupyter Notebook...

Reply:
Fixed